apcd_cms/src/client/src/hooks/admin/index.ts

```diff
@@ -111,11 +114,32 @@ export type ExceptionResult = {
   total_pages: number;
 };
 
+export type SubmitterUserRow = {
+  submitter_id: string;
+  user_id: string;
+  user_number: string;
+  created_at: string;
+  updated_at: string;
+  user_email: string;
+  user_name: string;
+  payor_code: string;
+};
+
+export type SubmitterUserResult = {
+  // selected_status: string;
```

Comment:
can be removed

Reply:
Remove the commented-out portions or _everything_ under `export type SubmitterUserResult`?

Reply:
no, just the commented out code, just unnecessary:
```
// selected_status: string;
```

Reply:
Done and done!